api/test/unit/conftest.py

```diff
@@ -1,40 +1,6 @@
-from dataclasses import dataclass
```

Comment:
All of these removed lines are just moved to api/test/fixtures/media_type_config.py